web/src/beta/components/Popup/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { ReactNode, RefObject, useEffect, useRef, useCallback } from "react";
+import FocusLock from "react-focus-lock";
```

Comment:
Can you explain why we need this? (BTW I don't think it is working the way the github page for this dependency says it should)
You can explain this to me on Slack in Japanese if its easier! 👍🏻 
I feel like we don't need this, and I'd prefer not adding a new dependency unless its VERY necessary